src/rgw/rgw_period_history.h

```diff
@@ -11,9 +11,9 @@
 #include "include/assert.h"
 #include "include/types.h"
 
-namespace bi = boost::intrusive;
+#include "rgw_rados.h"
```

Comment:
I was expecting a pull request where this line was the only change. Do the rest of the source files really need to mess with include order, after this line is added?